cric-predict/src/Components/LoginForm.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import '../Stylesheets/App.css';
+import { Row, Col, Form, Button } from 'react-bootstrap';
+import axios from 'axios';
+
+
+class LoginFormComponent extends React.Component {
```

Comment:
again better name would be `LoginComponent` and the file name `Login.js` .. it doesn't have to convey what markup it'd render